panel/src/components/View/Menu.vue

```diff
@@ -53,6 +54,12 @@ export default {
 	computed: {
 		menus() {
 			return this.$panel.menu.entries.split("-");
+		},
+		canSearch() {
+			return (
+				this.$panel.permissions.access['site'] === true ||
+				this.$panel.permissions.access['users']
```

Comment:
Same as with the PHP search view, this cannot be tied to just the site and users. Rather should be something like checking the length of `panel.searches`